scheduler/cloudformation/cloudformation.go

```diff
@@ -257,22 +261,109 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 			Parameters: parameters,
 			// TODO: Update Go client
 			// Tags:         tags,
-		}, done, ss); err != nil {
+		}, output, ss); err != nil {
 			return err
 		}
 	} else {
 		return fmt.Errorf("error describing stack: %v", err)
 	}
 
 	if ss != nil {
-		if err := <-done; err != nil {
-			return err
+		if err := s.waitUntilStable(ctx, output, ss); err != nil {
+			logger.Warn(ctx, fmt.Sprintf("error waiting for submit to stabilize: %v", err))
 		}
 	}
+	return nil
+}
 
+func (s *Scheduler) waitUntilStable(ctx context.Context, o chan stackOperationOutput, ss scheduler.StatusStream) error {
+	output := <-o
```

Comment:
yea my logic with that was you wouldn't get those if you hadn't specified a statusStream (since we don't wait) so they should just be warnings. but i agree it should be an error